scipy/stats/__init__.py

```diff
@@ -194,11 +194,27 @@
    zipf                     -- Zipf (Zeta)
    zipfian                  -- Zipfian
 
+Censored data class
+-------------------
+
+The univariate continuous distributions can be fit to censored data
+by maximum likelihood estimation.  Censored data is represented with
+instances of the `CensoredData` class.
+
+.. autosummary::
+   :toctree: generated/
+
+   CensoredData
```

Comment:
For some reason this is rendering like:
<img width="456" alt="image" src="https://user-images.githubusercontent.com/6570539/112406087-bc968780-8cd0-11eb-8536-876f81340d2d.png">

I imagine that will be a problem wherever this ends up.

Reply:
What browser are you using?  I get that messy overlap with quite a few objects (e.g. `rv_histogram`, `cumfreq`, `relfreq`, `weighedtau`) when I use an old version of Safari.  With a recent Firefox, it looks fine.

Reply:
That's the problem. Never mind.